modules/phase_field/src/vectorpostprocessors/GrainForcesPostprocessor.C

```diff
@@ -22,9 +22,15 @@ GrainForcesPostprocessor::GrainForcesPostprocessor(const InputParameters & param
     _grain_force_torque(getUserObject<GrainForceAndTorqueInterface>("grain_force")),
     _grain_forces(_grain_force_torque.getForceValues()),
     _grain_torques(_grain_force_torque.getTorqueValues()),
-    _grain_num(_grain_forces.size())
+    _grain_num(0)
 {
-  _grain_force_torque_vector.resize(_grain_num*12);
+}
+
+void
+GrainForcesPostprocessor::initialize()
+{
+  _grain_num = _grain_forces.size();
+  _grain_force_torque_vector.resize(_grain_num * 12);
```

Comment:
Ok, ammended. @SudiptaBiswas please note that I cut the output vector size in half!